examples/flask/src/charm.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Flask Charm service."""
+
+import logging
+import typing
+
+import ops
+
+import xiilib.flask
+
+logger = logging.getLogger(__name__)
+
+
+class FlaskCharm(xiilib.flask.Charm):
+    """Flask Charm service."""
+
+    def __init__(self, *args: typing.Any) -> None:
+        """Initialize the instance.
+
+        Args:
+            args: passthrough to CharmBase.
+        """
+        super().__init__(*args)
+
+
+if __name__ == "__main__":  # pragma: nocover
+    ops.main.main(FlaskCharm)
```

Comment:
hmm this might be a problem since we have to depend on `ops` so essentially we need to pin the major version of `ops`. Any thoughts @benhoyt? Also, @gregory-schiano

Reply:
ah yeh you are right, so it should be ok